cf/commands/buildpack/delete_buildpack.go

```diff
@@ -68,15 +78,22 @@ func (cmd *DeleteBuildpack) Execute(c flags.FlagContext) error {
 		}
 	}
 
-	cmd.ui.Say(T("Deleting buildpack {{.BuildpackName}}...", map[string]interface{}{"BuildpackName": terminal.EntityNameColor(buildpackName)}))
-	buildpack, err := cmd.buildpackRepo.FindByName(buildpackName)
+	if stack == "" {
+		buildpack, err = cmd.buildpackRepo.FindByName(buildpackName)
+		cmd.ui.Say(T("Deleting buildpack {{.BuildpackName}}...", map[string]interface{}{"BuildpackName": terminal.EntityNameColor(buildpackName)}))
+	} else {
+		buildpack, err = cmd.buildpackRepo.FindByNameAndStack(buildpackName, stack)
+		cmd.ui.Say(T("Deleting buildpack {{.BuildpackName}} with stack {{.Stack}}...", map[string]interface{}{"BuildpackName": terminal.EntityNameColor(buildpackName), "Stack": terminal.EntityNameColor(stack)}))
+	}
 
 	switch err.(type) {
 	case nil: //do nothing
 	case *errors.ModelNotFoundError:
 		cmd.ui.Ok()
 		cmd.ui.Warn(T("Buildpack {{.BuildpackName}} does not exist.", map[string]interface{}{"BuildpackName": buildpackName}))
```

Comment:
Update this error message to include the stack if it is provided; there may be a buildpack with the given name but a different stack.

Reply:
⏭will update error messages in next PR.